src/test/kotlin/interactor/GetCityHasCheapestInternetConnectionInteractorTest.kt

```diff
@@ -1,32 +1,60 @@
 package interactor
 
-import data.FakeDataSource
-import data.TestCase
+import data.CheapestInternetFakeData
+import data.EmptyFakeData
+import data.InvalidFakeData
 import org.junit.jupiter.api.*
 import org.junit.jupiter.api.Assertions.*
+import org.junit.jupiter.api.function.Executable
+
 
 
 @TestInstance(TestInstance.Lifecycle.PER_CLASS)
 internal class GetCityHasCheapestInternetConnectionInteractorTest {
 
     private lateinit var cityHasCheapestInternet: GetCityHasCheapestInternetConnectionInteractor
-    private lateinit var fakeData: FakeDataSource
+
+    private lateinit var fakeData: CheapestInternetFakeData
```

Comment:
You can put all this fake data in the same class and make this class singleton so you can change the type of fake data you like.
Check the way that  I use it with my case